consensus/miner_signalled_consensus.md

```diff
@@ -0,0 +1,247 @@
+# Miner Signalled Consensus Upgrade
```

Comment:
In general I feel like this document is not so much about the miner signalling and more about how a node deals with the signals. It might help to clear that up?